libs/entry-components/styles/partials/vendors/overrides/redactor/_general.scss

```diff
@@ -0,0 +1,38 @@
+@use 'sass:map';
+
+$material-text-color: rgb(0 0 0 / .87);
+$material-border: rgb(0 0 0 / .38);
+$material-border-radius: 2px;
+$material-icon-color: rgb(0 0 0 / .54);
+$material-checkbox-width: 18px;
+$material-checkbox-height: 18px;
+
+$redactor-todoitem-selector: '.rx-editor [data-rx-type = 'todoitem']';
+
+.enigmatry-redactor-content {
+	.rx-editor, .rx-content {
+		color: $material-text-color;
+	}
+
+	#{$redactor-todoitem-selector} {
+		input {
+			width: $material-checkbox-width;
+			height: $material-checkbox-height;
+		}
+
+		input::before, input:checked::before {
```

Comment:
if this is related to checkbox and input, then it should be in the forms file, not in general